Domains/4-Application/Connectors/CivilInfrastructureFramework/CifBridge.ecschema.xml

```diff
@@ -959,7 +959,7 @@
         <ECProperty propertyName="CrossFrameComponentHolder_CrossFrameGroupName" typeName="string" displayLabel="Cross-Frame Group" category="ComponentProperties_Component_Properties" priority="299997"/>
         <ECProperty propertyName="CrossFrameComponentHolder_Material" typeName="string" displayLabel="Material" category="ComponentProperties_Component_Properties" priority="299995"/>
         <ECProperty propertyName="CrossFrameComponentHolder_Volume" typeName="double" displayLabel="Volume" category="ComponentProperties_Component_Properties" priority="299977" kindOfQuantity="cifu:VOLUME"/>
-        <ECProperty propertyName="CrossFrameComponentHolder_Weight" typeName="double" displayLabel="Weight" category="ComponentProperties_Component_Properties" priority="299976" kindOfQuantity="cifu:LENGTH"/>
+        <ECProperty propertyName="CrossFrameComponentHolder_Weight" typeName="double" displayLabel="Weight" category="ComponentProperties_Component_Properties" priority="299976" kindOfQuantity="cifu:MASS"/>
```

Comment:
Because the kindOfQuantity is changed for an existing property AND the new kindOfQuantity has a different persistence unit than the old kindOfQuantity, you will also need a custom attribute here for schema update purposes. This ECProperty line should look like this:

    <ECProperty propertyName="CrossFrameComponentHolder_Weight" typeName="double" displayLabel="Weight" category="ComponentProperties_Component_Properties" priority="299976" kindOfQuantity="cifu:MASS">
        <ECCustomAttributes>
            <AllowUnitChange xmlns="SchemaUpgradeCustomAttributes.01.00.00">
                <From>u:M</From>
                <To>u:KG</To>
            </AllowUnitChange>
        </ECCustomAttributes>
    </ECProperty>